src/backend/revoker/scanAttestations.test.ts

```diff
@@ -71,13 +73,17 @@ describe('scanAttestations', () => {
     const owner = 'did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY';
     const createdAt = new Date('2020-01-01');
 
+    const attesterDidAsAccountHex = u8aToHex(
+      Utils.Crypto.decodeAddress(owner.split(':')[2]),
```

Comment:
I think there’s an API call to get the address part from the DID, isn’t there?

Reply:
Yes, there is one. But I thought it would be an overkill. Here is easier to follow.

Reply:
And also fragile and error-prone

Reply:
How about declaring the owner like this instead? 

`const owner = '4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY';`

Reply:
Let's overkill it! Why not? 

`Kilt.Did.toChain(owner)`

Reply:
Owner DID is also used in the check in this test. I guess I don’t mind hard-coding the address in one place and the DID in another place. I only mind building the DIDs from parts manually, since the way they are built might change in the future.

Reply:
```
   const attesterDidAsAccountHex = u8aToHex(
      Utils.Crypto.decodeAddress(Did.toChain(owner)),
    );
```

On [d79a6a4](https://github.com/BTE-Trusted-Entity/socialkyc.io/pull/2012/commits/d79a6a4625cde8c89853a71d9688d2e5600e3b3a)

Reply:
I hope the DIDs don't change in the future. But it's always better to be over-prepared than under-prepared.